package.json

```diff
@@ -17,7 +17,7 @@
     "generateFlowTypes": "./packages/components/generateFlowTypes.sh",
     "lint": "yarn run lint:styles && yarn run lint:scripts",
     "lint:fix": "yarn run lint:styles:fix && yarn run lint:scripts:fix",
-    "lint:styles": "stylelint --ignore-path .gitignore packages/**/*.{css,js,jsx,ts,tsx}",
```

Comment:
There's no reason to stylelint the JS files, right?

On the latest version it completely explodes on them.